packages/deployments/contracts/contracts_forge/core/connext/libraries/AmplificationUtils.t.sol

```diff
@@ -0,0 +1,118 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.8.14;
+
+import "../../../../contracts/core/connext/libraries/SwapUtils.sol";
+import "../facets/FacetHelper.sol";
+
+contract AmplificationUtilsTest is FacetHelper {
+  // ============ Storage ============
+  SwapUtils.Swap swap;
+
+  // ============ Events ============
+  event RampA(uint256 oldA, uint256 newA, uint256 initialTime, uint256 futureTime);
+  event StopRampA(uint256 currentA, uint256 time);
+
+  // ============ Setup ============
+  function setUp() public {
+    IERC20[] memory _pooledTokens = new IERC20[](1);
+    _pooledTokens[0] = IERC20(_adopted);
```

Comment:
It will be better to create pool with 2 or more tokens. Swap Pool with 1 token. lol

Reply:
good catch!